src/controlflow/agents/agent.py

```diff
@@ -289,7 +289,9 @@ def _run_model(
         tools = as_tools(self.get_tools() + tools)
         model = self.get_model(tools=tools)
 
-        logger.debug(f"Running model {model} for agent {self.name} with tools {tools}")
+        logger.debug(
+            f"Running model {controlflow.settings.llm_model} for agent {self.name} with tools {[t.name for t in tools]!r}"
```

Comment:
I think this is actually not correct for the model -- this will log the global setting, not the agent's own setting (if it was overridden)